plugins/cli.py

```diff
@@ -0,0 +1,33 @@
+import argparse as ap
+
+from .thread import ProcessThread
+from .options import AUTO_FILL_OPT, AUTO_TRANSLATE_OPT, EXCLUDE_DNP_OPT, EXTEND_EDGE_CUT_OPT, ALTERNATIVE_EDGE_CUT_OPT, EXTRA_LAYERS
```

Comment:
In sake of completeness, would you mind adding also 'ALL_ACTIVE_LAYERS_OPT' option? It was added in a recent PR.